src/backend/allocator/format.rs

```diff
@@ -66,6 +66,23 @@ macro_rules! format_tables {
             }
         }
 
+        /// Returns the transparent alternative of the specified format.
+        ///
+        /// If the format has an unused alpha channel, this may return the corresponding non-opaque format.
+        ///
+        /// Unknown formats will always return [`None`].
+        pub const fn get_transparent(
```

Comment:
Could you add a test to ensure we aren't breaking that behavior like the other tests for the format tables?

Reply:
Added https://github.com/Smithay/smithay/pull/935/files#diff-4defc011b2baa0cfeeadfc075ce3121157bcbf5ec99b70d725a130c9961d7035R384-R401